Tests/SafeDICoreTests/TypeDescriptionTests.swift

```diff
@@ -542,9 +542,9 @@ final class TypeDescriptionTests: XCTestCase {
         XCTAssertEqual(typeDescription.asSource, "() async throws -> ()")
     }
 
-    func test_asSource_whenDescribingAnUnknownCase_returnsTheProvidedStringWithWhitespaceStripped() {
```

Comment:
turns out we're not stripping leading whitespace. But that's honestly fine since we have no valid, compiling cases that result in an `.unknown` type.

This test wasn't previously testing the `.unknown` case, but rather the `.simple` case.